library/src/main/java/com/owncloud/android/lib/resources/status/OCCapability.kt

```diff
@@ -27,14 +27,15 @@ package com.owncloud.android.lib.resources.status
 /**
  * Contains data of the Capabilities for an account, from the Capabilities API
  */
-class OCCapability {
+class OCCapability(
```

Comment:
Why this change? If it's just for the tests, use `apply` instead. I don't see a reason to have only these three specific properties as constructor parameters.